assets/icons/file_icons/file_types.json

```diff
@@ -110,7 +110,8 @@
         "zsh_histfile": "terminal",
         "zsh_profile": "terminal",
         "zshenv": "terminal",
-        "zshrc": "terminal"
+        "zshrc": "terminal",
+        "vue": "vue"
```

Comment:
Can we keep this list sorted alphabetically?

Reply:
Okay, I have made the necessary changes